src/lib/styles/global/color-primitives.scss

```diff
@@ -0,0 +1,50 @@
+// Source: NNS Frontned dapp Figma
+:root {
+  // Purple
+  --purple-accent: #794ee7;
+  --purple-50: #fbfafc;
+  --purple-75: #f8f6fc;
+  --purple-100: #f1ecf8;
+  --purple-125: #e1d9f3;
+  --purple-150: #ddd4ec;
+  --purple-200: #d0c3ec;
+  --purple-250: #ab99d1;
+  --purple-400: #876ac8;
+  --purple-500: #7c5cc2;
+  --purple-600: #6f589e;
+  --purple-900: #382d50;
+
+  // Purple dark
+  --purple-dark-accent: #a36aff;
+  --purple-dark-50: #f03df5;
+  --purple-dark-75: #e8e1f2;
+  --purple-dark-100: #c7badb;
+  --purple-dark-200: #b8a1de;
+  --purple-dark-400: #483666;
+  --purple-dark-450: #3d2a5c;
+  --purple-dark-500: #372652;
+  --purple-dark-550: #31204c;
+  --purple-dark-600: #2d1a4a;
+  --purple-dark-650: #291942;
+  --purple-dark-900: #24143d;
+
+  // Accessory
+  --fade: rgba(36, 20, 61, 0.6); // #24143d 60%
```

Comment:
What is this colour for? I didn't find it in Figma.

Reply:
If there will be some derived colours, maybe `--fade` should live there?

Reply:
`rgba(var(--purple-dark-900), 0.6)`?

Reply:
I don't know, but it's a good question. I'll ask it to Artem and then change based on the feedback.

The derived colors will be in each theme. I will ask him where we should put those. I agree that maybe this one belongs there if it's a derived color.